test/DataStoreInMemoryImplementation.java

```diff
@@ -0,0 +1,22 @@
+package datastoreapi;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+public class DataStoreInMemoryImplementation {
+	private Input input;
+	private Output output;
+		

```

Comment:
For this `Input` and `Output`, they can use your `InputRequest` and `OutputResponse` classes from this PR: https://github.com/EDOLK/CPS353-Project/pull/16/files